torchgeo/datamodules/eurosat.py

```diff
@@ -67,11 +67,11 @@
     'B06': 861.18399006,
     'B07': 1086.63139075,
     'B08': 1117.98170791,
-    'B8A': 404.91978886,
-    'B09': 4.77584468,
-    'B10': 1002.58768311,
-    'B11': 761.30323499,
-    'B12': 1231.58581042,
+    'B09': 404.91978886,
+    'B10': 4.77584468,
+    'B11': 1002.58768311,
+    'B12': 761.30323499,
+    'B8A': 1231.58581042,
```

Comment:
These don't need to be moved around, this is a dictionary

Reply:
True, but it's better to keep them in the same order as everything else.